### DISCUSSION TITLE
The Discord Invite Is Invalid

### DISCUSSION THREAD

**Discussion by slrdur** [IrisShaders/Iris#1213] [Q&A] [answered]
Hey, I looked on the website, and twitter page, but the discord invite is invalid. Does anyone have one?

**Comment by NoComment1105** [comment, marked as answer]
The invite should not be invalid. Please try this one, https://discord.gg/jQJnav2jPu

**Reply by slrdur** [reply]
> The invite should not be invalid. Please try this one, https://discord.gg/jQJnav2jPu

I have no idea why it wasn't working a moment ago. It's all good now. Thanks!
